### PR TITLE
cleanup of docker tag command code

### PR DESCRIPTION
**Pull request by shishir-a412ed**

### CODE REVIEW

File: graph/service.go [hunk 1]
```diff
@@ -12,8 +12,7 @@ import (
 func (s *TagStore) Install(eng *engine.Engine) error {
 	for name, handler := range map[string]engine.Handler{
 		"image_set":      s.CmdSet,
-		"image_tag":      s.CmdTag,
-		"tag":            s.CmdTagLegacy, // FIXME merge with "image_tag"
+		"tag":            s.CmdTagLegacy,
```
Comment (LK4D4) [inline graph/service.go]: Maybe we should rename it to CmdTag then?

Comment (shishir-a412ed) [inline graph/service.go]: yeah I was just thinking about it :) ... but had already pushed the code.
I ll fix it.

**Comment by shishir-a412ed** [comment]
@LK4D4 I have changed it to CmdTag.

**Comment by crosbymichael** [comment]
LGTM

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**